Convex_hull_3/test/Convex_hull_3/test_ch_3_ambiguity.cpp

```diff
@@ -0,0 +1,32 @@
+#include <CGAL/Exact_predicates_inexact_constructions_kernel.h>
+#include <CGAL/Polyhedron_3.h>
+#include <CGAL/Surface_mesh.h>
+#include <CGAL/convex_hull_3.h>
+#include <vector>
+#include <fstream>
+
+
+typedef CGAL::Exact_predicates_inexact_constructions_kernel  K;
+typedef CGAL::Polyhedron_3<K>                     Polyhedron_3;
+typedef K::Point_3                                Point_3;
+typedef CGAL::Surface_mesh<Point_3>               Surface_mesh;
+
+
+int main(int argc, char* argv[])
+{
+  std::ifstream in( (argc>1)? argv[1] : "data/cross.off");
+  
+  Surface_mesh poly;
+  if(!(in >> poly))
+  {
+    std::cerr<<"Could not find input file."<<std::endl;
+    return 1;
+  }
+  
+  Surface_mesh chull;
+  // compute convex hull
+  auto np = CGAL::parameters::all_default();
+  CGAL::convex_hull_3(poly, chull, np);
```

Comment:
Just use a single call

Reply:
If I use a single test, then there won't be any ambiguity to test.

Reply:
`CGAL::convex_hull_3(poly, chull, parameters::all_default());` is the point

Reply:
Yes, but called like this there was no ambiguity for some reason (@lrineau explained it to me at the time but I forgot :/. The call with `auto` triggered the error this test is supposed to check, so I cannot just remove it, or else the test becomes useless.